.github/workflows/PRLabeling.yml

```
@@ -15,22 +15,55 @@
-      - name: Label PR
-        uses: actions-ecosystem/action-add-labels@v1
-        with:
-          labels: ${{ steps.pr_details.outputs.label }}
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          # Save label and color to environment variables
+          echo "LABEL=$LABEL" >> $GITHUB_ENV
+          echo "COLOR=$COLOR" >> $GITHUB_ENV
+
+      - name: Add Label with Color
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC1078:warning:5:6: Did you forget to close this single quoted string?

(shellcheck)